frontend/packages/integration-tests-cypress/support/index.ts

```diff
@@ -40,9 +41,17 @@ after(() => {
 
 export const checkErrors = () =>
   cy.window().then((win) => {
-    if (win.windowError) {
-      throw new Error(`window/js runtime error detected: ${win.windowError}`);
-    }
+    const assertMsg =
+      win.windowError !== null
+        ? `window error detected: ${(win.windowError as PromiseRejectionEvent).reason ||
+            (win.windowError as Error).message} ${
+            (win.windowError as Error).stack
+              ? (win.windowError as Error).stack.replace(/\\n/g, '\n')
+              : ''
+          }`
+        : 'no window error detected';
+
+    assert.isTrue(win.windowError === null, assertMsg);
```

Comment:
Don't check specifically for null. You want to handle undefined as well.

Reply:
fixed.